src/components/UI/Buttons/IconButton.vue

```diff
@@ -54,6 +57,12 @@ export default {
       type: Boolean,
       default: false,
     },
+    isStackVertically: {
+      // whether to stack the icon and title vertically
+      // instead of horizontally
+      type: Boolean,
+      default: false,
+    },
```

Comment:
better would be: orientation as input - horizontal or vertical and `isStackedVertically` - computed property